WordPress/src/test/java/org/wordpress/android/viewmodel/pages/PagesViewModelTest.kt

```diff
@@ -215,6 +219,24 @@ class PagesViewModelTest {
         verify(uploadStarter, times(2)).queueUploadFromSite(site)
     }
 
+    @Test
+    fun `postUploadAction invoked on edit post activity result`() = test {
+        // Given
+        val localPageId = 999
+        val intent = mock<Intent>()
+        val postModel = PostModel().apply { setStatus(PostStatus.PUBLISHED.toString()) }
+        val siteModel = SiteModel()
```

Comment:
Hi Jirka! Quick question. Why did you use a different `SiteModel` here instead of using the `site` that's already created? Let me know your thoughts. Thanks 😄

Reply:
Hi Joel :). 

> Why did you use a different SiteModel here instead of using the site that's already created?

:D Simple answer, because I overlooked it:P. Fixed in a3bf1776a423d8b30d6e0211e180608a032f4970

Reply:
Awesome 😉